INN/umbrella-borderzine
# 66 hide homepage excerpts mobile

## Changes

This pull request makes the following changes:

- Below 768px:
    - aligns article thumbnails to the right
    - adds 10px margin-left so headlines don't run into the thumbnails
    - sets thumbnail max-width to 110px to match the thumbnail size on the homepage featured section at the top of the page
- Moves the images in the homepage featured section to be level with the headlines, so that the headlines and images in this area match the headlines and images elsewhere on the homepage

Here's what it looks like at a typical phone size:

![Screenshot_2019-10-03 Borderzine - Reporting Across Fronteras](https://user-images.githubusercontent.com/1754187/66170464-53b9dc00-e612-11e9-85af-b63c8af44191.png)


Specifically as regards the levelling of the images and headlines:

Before:
<img width="1184" alt="Screen Shot 2019-10-03 at 19 12 30 " src="https://user-images.githubusercontent.com/1754187/66170340-e5751980-e611-11e9-9308-521f1d5ce3b8.png">

After:
<img width="1184" alt="Screen Shot 2019-10-03 at 19 12 19 " src="https://user-images.githubusercontent.com/1754187/66170365-fa51ad00-e611-11e9-96a7-53a8103a9a23.png">

## Why



Resolves #66 

## Testing/Questions

Features that this PR affects:

- homepage


Questions that need to be answered before merging:


Steps to test this PR:

1. check out branch
2. visit homepage
3. ![resizing](https://gifs.benlk.com/resizing.gif)
